numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -3,14 +3,50 @@
 import os
 from collections import namedtuple
 import platform
-
+import site
+from pathlib import Path
 from numba.core.config import IS_WIN32
-from numba.misc.findlib import find_lib, find_file
+from numba.misc.findlib import find_lib
 from numba import config
+import ctypes
 
+IS_LINUX = sys.platform == "linux"
```

Comment:
For all practical purposes, this is equivalent to `not IS_WIN32`

To make this code easier to read, I think it would be better to consistently use `IS_WIN32` (and not introduce this extra variable).